util/util_test.go

```diff
@@ -29,3 +29,143 @@ func mockParam(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(200)
 }
+
+func TestUserCache(t *testing.T) {
```

Comment:
please put this test in to cache_test.go instead (and shout out for actually adding tests! :1st_place_medal: )